File: slimadapter/src/main/java/com/jhj/slimadapter/callback/ItemViewBind.java
package com.jhj.slimadapter.callback;

import android.support.annotation.NonNull;

import com.jhj.slimadapter.SlimAdapter;
import com.jhj.slimadapter.holder.ViewInjector;

/**
 * 实现布局方法、数据、数据位置回调
 * <p>
 * Created by jhj on 18-10-13.
 */

public interface ItemViewBind<D> {

    void convert(SlimAdapter adapter, @NonNull ViewInjector injector, D bean, int position);

}

File: slimadapterkt/src/main/java/com/zgdj/slimadapterkt/callback/ItemViewBind.kt
package com.zgdj.slimadapterkt.callback

import com.zgdj.slimadapterkt.SlimAdapter
import com.zgdj.slimadapterkt.holder.ViewInjector


/**
 * 实现布局方法、数据、数据位置回调
 *
 *
 * Created by jhj on 18-10-13.
 */

interface ItemViewBind<in D> {

    fun convert(adapter: SlimAdapter, injector: ViewInjector, bean: D, position: Int)

}
